html/themes/custom/common_design_subtheme/common_design_subtheme.theme

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @file
+ * Template overrides, preprocess, and alter hooks for the Common Design theme.
+ */
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function common_design_subtheme_preprocess_page(&$variables) {
+  // Get results page path - default to 'results'.
+  $variables['results_page_path'] = \Drupal::config('ocha_search.settings')->get('site_results_page_path') ?? 'results';
+}
```

Comment:
This can be moved to the base theme as well

Reply:
@attiks  I think the question is if we can move it to https://github.com/UN-OCHA/ocha_search - and the template too - it's only used to change the 'action' in the form: https://github.com/UN-OCHA/common-design-site/blob/8242a8697afd019c0f2d57698c44571ec863cc75/html/themes/custom/common_design_subtheme/templates/cd/cd-header/cd-search.html.twig#L6 - but the config setting doesn't exist if the module isn't enabled.

Reply:
Yes, both of them can be moved to https://github.com/UN-OCHA/ocha_search